sdk/azidentity/interactive_browser_credential.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azidentity
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"strings"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/pkg/browser"
+)
+
+// InteractiveBrowserCredentialOptions can be used when providing additional credential information, such as a client secret.
+// Also use these options to modify the default pipeline behavior through the TokenCredentialOptions.
+type InteractiveBrowserCredentialOptions struct {
+	ClientSecret *string                 // Gets the client secret that was generated for the App Registration used to authenticate the client. Only applies for web apps.
```

Comment:
Doc comments should go above the field in the same format as the outer doc comment, i.e. `// ClientSecret ...`.

Reply:
Done